web/components/contract/contract-info-dialog.tsx

```diff
@@ -113,14 +113,9 @@ export function ContractInfoDialog(props: { contract: Contract; bets: Bet[] }) {
           <TagsInput contract={contract} />
           <div />
 
-          {contract.mechanism === 'cpmm-1' &&
-            !contract.resolution &&
-            (!closeTime || closeTime > Date.now()) && (
-              <>
-                <div className="">Add liquidity</div>
-                <AddLiquidityPanel contract={contract} />
-              </>
-            )}
+          {contract.mechanism === 'cpmm-1' && !contract.resolution && (
```

Comment:
What about closed markets? You can still add / withdraw liquidity?

Reply:
yes!